ci-operator/step-registry/cucushift/upgrade/eus/unpause-worker-mcp/cucushift-upgrade-eus-unpause-worker-mcp-ref.yaml

```diff
@@ -8,6 +8,13 @@ ref:
   grace_period: 10m
   commands: cucushift-upgrade-eus-unpause-worker-mcp-commands.sh
   cli: latest
+  env:
+  - name: PAUSED_MCP_LABEL
+    default: "worker"
+    documentation: "the label of mcp which will have been paused"
+  - name: NORMAL_MCP_LABEL
```

Comment:
What is the purpose of introducing this ENV, what is its difference with the default `worker` mcp ?

Reply:
because in some job we don't know how many worker nodes we have, for example in baremetalds-ipi-ovn-ipv4-fips-f14, we have 3 worker nodes but we only labeled 2 of them (1 with mcpfoo, 1 with mcpbar), so when we want to remove the label, I need the label to find out the node.